### DISCUSSION TITLE
Project - Why does GDP Matter? [long_run_growth]

### DISCUSSION THREAD

**Discussion by mmcky** [QuantEcon/reading-group-2024#4] [Ideas]
Q: Why does GDP matter? What does it correlate with (HDI, Health outcomes)

Presentations next week on some correlations of interest and how we can start to answer the `Why does GDP matter?` question

**Comment by HumphreyYang** [comment]
Hi @pgrosser1 @Jiarui-ZH @mbek0605,

Thanks for the great discussion last week. Please feel free to post your thoughts and findings : )

(By the way, please feel free to post new topics related to this lecture series that interest you!)
